src/routes/apiDirectory.js

```diff
@@ -71,6 +73,10 @@ router.use('/feeds', feedRouter);
 router.use('/resources', resourcesRouter);
 router.use('/goal-templates', goalTemplatesRouter);
 
+if (isTestingOrCI()) {
+  router.use('/testing', testingRouter);
```

Comment:
**Not** a request for change, but a comment - this route existing, even as locked down as it is (and I acknowledge it would be tricky for some kind of malicious actor to access it), makes me a little nervous. 

Was there a specific use case we needed an API route for that the functions wouldn't be enough for?

Reply:
Yes the e2e test need this most

Reply:
@thewatermethod, In talking with @nvms we came up with an idea to shift this over to a distinct nodejs app container that would only live in CI and be excluded from ever being deployed to production. as even further protection. Using this method we could add more routes in the future to allow e2e test to better prep the env by making changes to the DB directly without the need to add routes for testing to the main app.

Reply:
thinking out loud here, it sounds like what we should do is...

- create a very small express app that has these destructive routes (e.g. `/reseed`)
- the existing test job in CI spawns a new container that runs this express app
- this express app doesn't get sent to CF because it's included in `.cfignore`